images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN oryx prep --skip-detection --platforms-and-versions nodejs=12 \
     && . /tmp/build/__goVersions.sh \
     && downloadedFileName="go${GO_VERSION}.linux-amd64.tar.gz" \
     && curl -SLsO https://golang.org/dl/$downloadedFileName \
-    && tar -C /usr/local -xzf $downloadedFileName \
+    && mkdir -p /usr/local \
+    && tar -xzf $downloadedFileName -C /usr/local \
```

Comment:
this tar command was initially failing so I added a mkdir command before it to make sure the directory exists where we untar the components